index.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Enhance Your Skills GLOBAL Summit 2022</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav
+      class="navbar nav1 navbar-expand-sm bg-dark navbar-dark justify-content-end py-0 p-3"
+    >
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/linke.png" alt="linkedin" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link navlink" href="#">English</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link navlink" href="#">My Page</a>
+        </li>
+        <li class="nav-item navLogout">
+          <a class="nav-link navlink" href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+
+    <!-- // navbar Section // -->
+
+    <nav class="navbar navbar-expand-sm bg-light navbar-light">
+      <div class="container">
+        <a class="navbar-brand" href="./index.html">
+          <img src="./images/Enha (1).png" alt="logo" id="logoMain" />
```

Comment:
- [ OPTIONAL ] I think your logo doesn't take to the main page but as required it should take the user to the main page so kindly a the right link and make sure it takes you to the main page 💯💯

Reply:
Thanks for the suggestion. Now it is working.